src/components/cards/card.tsx

```diff
@@ -1,6 +1,6 @@
-import * as React from 'react';
-import './card.css';
-
-export const Card = (props: React.PropsWithChildren<{}>) => (
-    <div className="card">{props.children}</div>
-)
\ No newline at end of file
+import * as React from "react";
+import styles from "./card.module.css";
+
+export const Card = (props: React.PropsWithChildren<{}>) => (
+  <div className={styles.card}>{props.children}</div>
```

Comment:
I'll only point this out once, but using css modules allows us to compose class names like this, which is great for avoiding specificity issues